packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -56,7 +54,10 @@ export interface Props {
 
 	onMoreOptions?: () => void;
 
-	recordAnalytics?: ( event: RecordTrainTracksEventProps ) => void;
+	// TODO: How to use analytics type here?
+	// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+	// @ts-ignore
```

Comment:
@sirreal any thoughts on this todo?

Reply:
This is replied here. https://github.com/Automattic/wp-calypso/pull/42409#discussion_r428682775 
Resolving this conversation.